content/en/faqs/Is-Privacy-Badger-breaking-YouTube.md

```diff
@@ -0,0 +1,10 @@
+---
+question: How should I handle issues with Privacy Badger on YouTube? 
```

Comment:
Maybe: "Why aren't videos loading on YouTube? Why isn't Privacy Badger blocking ads on YouTube?"

Reply:
I was trying to create one question that captured both so that it made sense to have one FAQ instead of two separate FAQs. But I guess there's another [compound FAQ](https://privacybadger.org/#What-is-the-Privacy-Badger-license--Where-is-the-Privacy-Badger-source-code), so I'll use the language you're suggesting!